projects/website/src/releases/release-list.json

```diff
@@ -1,6 +1,9 @@
 {
-  "current": "13.0.2",
+  "current": "13.0.3+",
```

Comment:
Why do we have `+` sign - I think that we don't need to have one. And just add release file for every release as we always did but keep the link pointing to the Release in Github